File: src/main/java/grp/faq/dao/PraiseMapper.java
package grp.faq.dao;

import grp.faq.entity.Praise;

public interface PraiseMapper {
    int deleteByPrimaryKey(Long cPraiseid);

    int insert(Praise record);

    int insertSelective(Praise record);

    Praise selectByPrimaryKey(Long cPraiseid);

    int updateByPrimaryKeySelective(Praise record);

    int updateByPrimaryKey(Praise record);
}

File: src/main/scala/grp/faq/dao/PraiseMapper.scala
package grp.faq.dao

import grp.faq.entity.Praise
import org.apache.ibatis.annotations.Param

trait PraiseMapper {
  def praise(@Param("type")praiseType: Int, @Param("id")id: Long, @Param("uid")uid: Long): Int
}